geo.txt

```diff
@@ -110,7 +110,9 @@ characterization by rotational spectroscopy and ab initio calculations. Phys Che
 Ag  I   2.5483(1)
 P...Ag  2.3488(20)
 P   H   1.4086
-H   P H 118.92 
+H   P H 118.92(b)
+
+(b) assumed
```

Comment:
I wrote a comment here a moment ago but can't find what I wrote anymore. 

The number in parentheses is the uncertainty in the last digit shown. Putting a letter there can be confusing and would make it harder for a computer program to parse.

I suggest putting a comment like: "The HPH angle of 118.92 degrees is an "assumed" angle.